addons/website/static/src/js/content/website_root.js

```diff
@@ -262,9 +262,9 @@ var WebsiteRoot = BodyManager.extend({
         })
         .fail(function (err, data) {
             return new Dialog(self, {
-                title: data.data ? data.data.arguments[0] : "",
+                title: data.data ? data.data.arguments[0] : err.data.name,
```

Comment:
```suggestion
                title: data.data ? data.data.arguments[0] : _t('An error occured.'),
```
Showing `odoo.exceptions.ValidationError` is really not a good UX for the enduser.
I would just show that generic message.